src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/NewHandlerTest.java

```diff
@@ -36,14 +36,14 @@
     @Override
     protected String getPath(String filename) throws IOException {
         return super.getPath("checks" + File.separator
-                + "indentation" + File.separator + filename);
+                + "indentation" + File.separator + "newhandler" + File.separator + filename);
     }
 
     @Test
     public void testInvalidLabel() throws Exception {
         final DefaultConfiguration checkConfig = createCheckConfig(IndentationCheck.class);
```

Comment:
@romani Should I continue with fix in same PR or open a new PR under the issue? Also when I made this fix there was one file left. Should I go ahead and remove that extra file as well under the same fix that would be made?